src/ConcordiumNetSdk/Types/BlockHash.cs

```diff
@@ -52,4 +57,34 @@ public Concordium.V2.BlockHash ToProto()
             Value = Google.Protobuf.ByteString.CopyFrom(this.GetBytes())
         };
     }
+
+    /// <summary>
+    /// Converts the block hash to a corresponding <see cref="BlockHashInput"/>
+    ///
+    /// This can be used as the input for class methods of <see cref="ConcordiumClient.RawClient"/>.
+    /// </summary>
+    public BlockHashInput ToBlockHashInput()
+    {
+        return new BlockHashInput() { Given = ToProto() };
+    }
+
+    /// <summary>
+    /// Returns a <see cref="BlockHashInput"/> which specifies the best block.
+    ///
+    /// This can be used as the input for class methods of <see cref="ConcordiumClient.RawClient"/>.
+    /// </summary>
+    public static BlockHashInput BestBlockHashInput()
+    {
+        return new BlockHashInput() { Best = new Empty() };
+    }
+
+    /// <summary>
+    /// Returns a <see cref="BlockHashInput"/> which specifies the last finalized block.
+    ///
+    /// This can be used as the input for class methods of <see cref="ConcordiumClient.RawClient"/>.
+    /// </summary>
+    public static BlockHashInput LastFinalBlockHashInput()
```

Comment:
Shouldn't we call this just LastFinal? Same for Best? Don't we call this as


```
BlockHashInput.LastFinal
```

Reply:
I can add classes `BlockHashInput` and `AccountIdentifierInput` if you want me to. Currently these are instantiated from `BlockHash` and `AccountAddress` 🤔 but it will probably me more correct to have these and then invoke `ToProto` with their instances as the receiver?

Reply:
It's perhaps better to scrap these for now and do `BlockHashInput() { Best = new Empty() }`. It doesn't really seem right to me to do it this way.